example/reinforcement-learning/a3c/README.md

```diff
@@ -7,6 +7,12 @@ The algorithm should be mostly correct. However I cannot reproduce the result in
 
 Note this is a generalization of the original algorithm since we use `batch_size` threads for each worker instead of the original 1 thread.
 
+## Prerequisites
+  - Install OpenAI Gym: `pip install gym`
+  - Install the Atari Env: `pip install gym[atari]`
+  - You may need to install flask: `pip install flask`
+  - You may have to install cv2: `pip install opencv-python`
+
```

Comment:
Instead of using these Pre-reqs, the example also worked by just installing rllab as here - http://rllab.readthedocs.io/en/latest/user/installation.html